### PR TITLE
Work on importing IMDB to new NDM

### PR DESCRIPTION
**Pull request by marcfor**

### CODE REVIEW

File: ckanext/stcndm/bin/get_unique_field_values.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    echo >&2 "$@"
+    exit 1
+}
+
+[ "$#" -eq 2 ] || die "2 arguments required, $# provided. arg1: organisation, arg2: field name"
+
+curl http://localhost:8983/solr/ndm/query?q=organization:$1\&rows=70000\&fl=$2 | \
+   jq '.response.docs' | \
+   jq "[.[].$2]" | \
+   jq 'unique'
```
Comment (wardi) [inline ckanext/stcndm/bin/get_unique_field_values.sh:12]: Cool! didn't know you could do this so easily with jq

File: ckanext/stcndm/bin/massage_imdb.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+content_type_list = []
+for line in run_command('curl http://127.0.0.1:5000/api/3/action/package_search?q=type:codeset&rows=100'.split()):
+    raw_codesets = json.loads(line)['result']['results']
+    content_type_list = []
+    for codeset in raw_codesets:
+        if codeset['codeset_type'] == 'content_type':
+            content_type_list.append({
+                'label': codeset['title'],
+                'value': codeset['codeset_value']
+            })
```
Comment (wardi) [inline ckanext/stcndm/bin/massage_imdb.py:44]: You could avoid shelling out to curl by using [ckanapi.RemoteCKAN](https://github.com/ckan/ckanapi#remoteckan) just like: https://github.com/open-data/ckanext-stcndm/blob/master/ckanext/stcndm/helpers.py#L248-L253

File: ckanext/stcndm/schemas/imdb.yaml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+dataset_fields:
+
+- field_name: archive_date
+  label:
+    en: Archive Date
+    fr: Date d'archivage
+  preset: date
+  form_snippet: date.html
+  display_snippet: date.html
```
Comment (wardi) [inline ckanext/stcndm/schemas/imdb.yaml:23]: These display_snippet and form_snippet values should already be provided by the date preset.

**Comment by wardi** [comment]
@marcfor If the scripts you've written are meant to be reusable they should have a parameter for the solr core or ckan instance being queried. Currently they're hard-coded to htttp://localhost:<port>...

File: ckanext/stcndm/schemas/imdb.yaml (hunk 3 above)
Comment (marcfor) [inline ckanext/stcndm/schemas/imdb.yaml:23]: OK, I've removed them.

File: ckanext/stcndm/bin/massage_imdb.py (hunk 2 above)
Comment (marcfor) [inline ckanext/stcndm/bin/massage_imdb.py:44]: Cool.  I've got that implemented now.

**Comment by marcfor** [comment]
I agree, the scripts to import data should take URLs as arguments.  I'll fix that before I ask someone to try them out.

The other scripts are just to help me explore the data.  I included them in the repository mostly as a backup.

**Pull request merged**